src/sas/qtgui/Perspectives/Fitting/ComplexConstraint.py

```diff
@@ -137,7 +137,7 @@ def setupTooltip(self):
         Tooltip for txtConstraint
         """
         p1 = self.tab_names[0] + ":" + self.cbParam1.currentText()
-        p2 = self.tab_names[1] if len(self.tab_names) != 1 else self.tab_names[0] +"."+self.cbParam2.currentText()
+        p2 = self.tab_names[1] +"."+self.cbParam2.currentText() if len(self.tab_names) != 1 else self.tab_names[0] +"."+self.cbParam2.currentText()
```

Comment:
This is probably a part of another commit?

Reply:
Yes, this fixes a failing unit test, which was causing the interpreter to exit and not perform the rest of the unit tests

Reply:
Interesting. On Windows it just marks the test fail without crash. This used to run, so must've been something I introduced (semi) recently... thanks for spotting this issue and fixing